airbyte-integrations/connectors/source-googleanalytics-singer/source_googleanalytics_singer/source.py

```diff
@@ -37,21 +39,104 @@ class GoogleAnalyticsSingerSource(BaseSingerSource):
     tap_cmd = "tap-google-analytics"
     tap_name = "Google Analytics API"
     api_error = Exception
+    _custom_reports_schema = {
```

Comment:
should this be in the python code or in its own JSON file?

Reply:
I think I can transfer this to the json file, no problem.